### DISCUSSION TITLE
Radio button group component bug

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3441] [Help: Figma] [answered]
hey, I found one issue with the Radio Button Group component in Figma. The problem is with selected radio item. It has 38px instead of 36px.

<img width="1041" alt="Zrzut ekranu 2023-08-24 o 14 52 02" src="https://github.com/twilio-labs/paste/assets/140410476/c7b1dfe0-a53c-4ed9-8483-681ba3736bf0">

**Comment by serifluous** [comment]
Hey @marektonik ! Thanks for calling this out. I fixed this in Paste Components—just waiting on Figma Support to help us resolve the publishing issue.

**Comment by serifluous** [comment, marked as answer]
The fix has just been published in our Figma library!
